api/src/main/java/org/pmiops/workbench/interceptors/InterceptorUtils.java

```diff
@@ -13,15 +13,14 @@ public class InterceptorUtils {
   private InterceptorUtils() {}
 
   public static Method getControllerMethod(HandlerMethod handlerMethod) {
-    // There's no concise way to find out what class implements the delegate interface, so instead
-    // depend on naming conventions. Essentially, this removes "Api" from the class name.
-    // If this becomes a bottleneck, consider caching the class mapping, or copying annotations
-    // from our implementation to the Swagger wrapper at startup (locally it takes <1ms).
-    Method apiControllerMethod = handlerMethod.getMethod();
-    String apiControllerName = apiControllerMethod.getDeclaringClass().getName();
+    String apiControllerName = handlerMethod.getBeanType().getName();
```

Comment:
The biggest change here was using getBeanType instead of getDeclaringClass for apiControllerName.

This became an issue, because OpenApi ApiDelegates now define their functions as default, so they are no longer declared in ApiControllers. This broke our assumption of getDeclaringClass(), because instead of returning the generated ApiControllers it was returning the generated Api.

Old Behavior:
![image](https://github.com/all-of-us/workbench/assets/24514630/d01956bf-f7c0-4484-82f8-5903cb45393d)

New Behavior:
![image](https://github.com/all-of-us/workbench/assets/24514630/960ef2b6-afeb-4df2-a789-e08877e65daa)